pkg/identityserver/user_registry.go

```diff
@@ -64,6 +64,7 @@ var (
 	errPasswordStrengthUppercase = errors.DefineInvalidArgument("password_strength_uppercase", "need at least `{n}` uppercase letter(s)")
 	errPasswordStrengthDigits    = errors.DefineInvalidArgument("password_strength_digits", "need at least `{n}` digit(s)")
 	errPasswordStrengthSpecial   = errors.DefineInvalidArgument("password_strength_special", "need at least `{n}` special character(s)")
+	errPermissionDenied          = errors.DefinePermissionDenied("permission_denied", "unauthorized request to restricted resource")
```

Comment:
This error is too generic to be in `user_registry.go`

Reply:
Could go with the `is.RequireAdmin` I suggested in my comment on `listUsers`. After that, make sure to run `mage go:messages`.